keyboards/keychron/c3_pro/readme.md

```diff
@@ -0,0 +1,21 @@
+# Keychron C3 Pro
+
+![Keychron C3 Pro](https://i.imgur.com/79sp3PH.jpg)
+
+A customizable 87% keyboard.
+
+* Keyboard Maintainer: [Keychron](https://github.com/keychron)
+* Hardware Supported: Keychron C3 Pro
+* Hardware Availability:[Keychron](https://github.com/keychron)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make keychron/c3_pro/ansi/red:default
+
+Flashing example for this keyboard:
+
+    make keychron/c3_pro/ansi/red:default:flash
+
+**Reset Key**: Hold down the key located at *K00*, commonly programmed as *Esc* while plugging in the keyboard.
```

Comment:
Is there a hardware reset button, too?  Normally, under the spacebar?

Reply:
Hi, yes, i added it.